index.html

```diff
@@ -3,14 +3,102 @@
 
 <head>
   <meta charset="UTF-8">
-  <title>css animations</title>
+  <title>Css - Animation</title>
   
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+        <link rel="stylesheet" href="css/search.css">
+        <link rel="stylesheet" href="css/popup.css">
+        <link rel="stylesheet" href="css/cart.css">
+        <link rel="stylesheet" href="css/index.css">
+        <link rel="stylesheet" href="css/submit.css">
+        <link rel="stylesheet" href="css/input.css">
+        <link rel="stylesheet prefetch" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
+
 </head>
 
 <body>
 
-<h1> CSS Animations </h1>
+	<div class="section-one">
+      <div class="search">
+      	<input type="search" class="search-box" placeholder="search" />
+     	 <span class="search-button">
+     	   <span class="search-icon"></span>
+     	 </span>
+	  </div>
+  </div>
+  
+  
+  <div class="">
+    <div class="flex">
+      <div class="share">
+        <a href="#0" class="toggle"><span>SHARE</span></a>
+        <button type="button" class="close">
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
+        <div class="content">
+          <div class="text">
+            <ul>
+              <li>Facebook <span>13</span></li>
+              <li>Twitter <span>18</span></li>
+              <li>Google+ <span>2</span></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+   </div>
+</div>
+
+  <div class="section-one">
+ 	 <button class="btn btn--add-to-cart">
+ 		 <svg class="btn__swiper" width="202" height="48px" version="1.1" xmlns="http://www.w3.org/2000/svg">
+  			  <path d="M 0 0
+				 L 202 0
+				 L 192 48
+				 L 0 48"
+			  	stroke="none">
+   			 </path>
+ 		 </svg>
+      <span class="btn__icon">
+        <i class="fa fa-shopping-cart fa-fw"></i>
+      </span>
+  	 <span class="btn__label">
+      Add to cart
+     </span>
+   </button>
+ </div>
+  
+  
+  <div class="">
+      <button id="submit">
+         	  <span class="icon"><i class="fa fa-check fa-fw"></i></span>
+         	  <span class="text">Submit
+              </span>
+              <span class="spinner">
+                <svg width="50" height="50">
+                  <circle r="23" cx="25" cy="25" />
+                </svg>
+          	 </span>
+    </button>
+    <a id="reset" href="#">Reset</a>

```

Comment:
add some styling to it, it has a color of white on white background, you can not see it :)

Reply:
done!